File: BookMyShowAPI/Data/DataModels/ReservedSeat.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Data.DataModels
{
    [Table("ReservedSeat")]
    public class ReservedSeat
    {
        [Column("Id")]
        [Key]
        public int Id { get; set; }
        [Column("SeatsNumber")]
        public int SeatNumber { get; set; }
        [Column("TheatreId")]
        public int TheatreId { get; set; }
    }
}

File: BookMyshowFrontend/bookMyShow/src/app/shared/models/reserved-seat.ts
export class ReservedSeat {
    id: number;
    seatNumber: number;
    theatreId: number;
    constructor(id: number, seatNumber: number, theatreId: number) {
        this.id = id
        this.seatNumber = seatNumber;
        this.theatreId = theatreId;
    }
}